main.js

```diff
@@ -1,4 +1,5 @@
-import UiBook from './UiBook.js';
+import UiBook from './modules/UiBook.js';
+import { DateTime } from './modules/luxon.js';
 
```

Comment:
- [ ] For this project you are supposed to name the entry point for your JavaScript code called `index.js`. Please change this file name to `index.js` from `main.js`. 

### See the image for reference

![image](https://user-images.githubusercontent.com/83803414/160464652-f00f4931-4ae9-4e3c-99ae-aaa0bad5a44f.png)